.refactored/sciserver-essentials/2.0/Dockerfile

```diff
@@ -0,0 +1,194 @@
+FROM dockerregistry:443/sciserver-jupyter:2.0
+
+# Python (Anaconda)
+
+RUN conda create -y --name py38 python=3.8 anaconda=2020.11 \
+	&& conda clean -y --all \
+	&& source activate py38 \
+	&& python -m ipykernel install --user --name py38 --display-name "Python 3.8 (py38)" \
+	&& /bin/bash -c 'printf "source activate py38\n" >> ~/.bashrc'
+
+
+# R
+
+USER root
+
+RUN yum -y install epel-release \
+	&& yum config-manager --set-enabled PowerTools \
+	&& yum -y install \
+		gcc-gfortran \
+		zeromq-devel \
+		zlib-devel \
+		readline-devel \
+		bzip2-devel \
+		xz-devel \
+		pcre-devel \
+		openssl-devel \
+		libcurl-devel \
+		libpng-devel \
+		libtiff-devel \
+		libjpeg-devel \
+		cairo-devel \
+		libXt-devel \
+		texinfo \
+		pandoc \
+		glibc-locale-source \
+		glibc-langpack-en \
+	&& yum clean all \
+	&& rm -rf /var/cache/yum
+
+USER idies
+
+ARG R_VERSION=4.0.3
+
+RUN curl -L https://cran.rstudio.com/src/base/R-4/R-$R_VERSION.tar.gz -o R.tar.gz \
+	&& tar -xzf R.tar.gz \
+	&& cd R-$R_VERSION \
+	&& ./configure --prefix=/home/idies/R --enable-R-shlib --with-x=no \
+	&& make -j`nproc` \
+	&& make install \
+	&& cd .. \
+	&& rm -rf R.tar.gz R-$R_VERSION
+
+ENV PATH /home/idies/R/bin:$PATH
+
+RUN Rscript -e "install.packages(c('Cairo', 'IRkernel'), repos='https://cloud.r-project.org/')" \
+	&& Rscript -e "IRkernel::installspec(displayname = 'R $R_VERSION')" \
+	&& /bin/bash -c $'printf "options(bitmapType=\'cairo\')\n" >> ~/.Rprofile'
+
+
+# SciScript Python
+
+RUN conda install -n py38 -y \
+		argcomplete \
+		chest \
+		configobj \
+		dill \
+		redis \
+		redis-py \
+		sphinx_rtd_theme \
+		traceback2 \
+		unittest2 \
+	&& conda install -n py38 -y -c conda-forge \
+		grequests \
+	&& conda clean -y --all
+
+RUN git clone https://github.com/sciserver/SciScript-Python.git \
+	&& touch /home/idies/keystone.token \
+	&& cd /home/idies/SciScript-Python \
+	&& git checkout sciserver-v2.0.13 \
+	&& cd py3 \
+	&& (source activate py38 && python setup.py install) \
+	&& rm /home/idies/keystone.token
+
+
+# SciScript R
+
+RUN Rscript -e "install.packages(c('bit64', 'jpeg', 'httr', 'jsonlite', 'data.table'), repos='https://cloud.r-project.org/')"
+
+RUN git clone https://github.com/sciserver/SciScript-R.git \
+	&& touch /home/idies/keystone.token \
+	&& cd /home/idies/SciScript-R \
+	&& git checkout sciserver-v2.0.13 \
+	&& cd .. \
+	&& R CMD build SciScript-R \
+	&& R CMD INSTALL SciServer*.tar.gz \
+	&& rm -rf /home/idies/SciServer*.tar.gz \
+	&& rm /home/idies/keystone.token
+
+
+# Mamba (fast package manager)
+
+RUN conda install -y -c conda-forge mamba \
+        && conda clean -y --all
+
+
+# Tensorflow
+
+RUN mamba install -n py38 -y tensorflow==2.3.0 \
```

Comment:
is it possible to get tensorflow 2.4+? Maybe not with conda, just checking

Reply:
Installing Tensorflow w/o conda was *really* painfull, last time I checked. Had to build it from source to make it work on the SciServer machines. Things might have improved since then, so I could check, but would really prefer to stick with the conda version.

Reply:
Yeah, makes sense. 2.3.0 is pretty good, and not that old (but will be quickly!) wouldn't bother too much with it now.